lib/Models/Workbench.ts

```diff
@@ -184,21 +184,19 @@ export default class Workbench {
           target &&
           GroupMixin.isMixedInto(target) &&
           !MappableMixin.isMixedInto(target) &&
-          !Chartable.is(target)
+          !ChartableMixin.isMixedInto(target)
         ) {
           this.remove(item);
         } else if (target) {
           return this.add(target);
         }
       }
 
+      if (CatalogMemberMixin.isMixedInto(item)) await item.loadMetadata();
+
       if (MappableMixin.isMixedInto(item)) {
         await item.loadMapItems();
       }
-
-      if (Chartable.is(item)) {
-        await item.loadChartItems();
-      }
```

Comment:
Does this now only work if a chartable item is also mappable? All of our chartables are currently mappable, but it's not guaranteed.

Reply:
I think this should read:
```ts
      if (MappableMixin.isMixedInto(item) || Chartable.is(item)) {
        await item.loadMapItems();
      }
```